website/source/docs/upgrade-specific.html.md

```diff
@@ -17,8 +17,17 @@ upgrade flow.
 
 ## Consul 1.7.0
 
-Consul 1.7.0 contains two major changes that may impact upgrades: 
-[stricter JSON decoding](#stricter-json-decoding) and [modified DNS outputs](#dns-ptr-record-output)
+Consul 1.7.0 contains three major changes that impact upgrades: 
+[stricter JSON decoding](#stricter-json-decoding), [modified DNS outputs](#dns-ptr-record-output), 
+and [backward-incompatible Session API changes](#session-api).
+
+### Session API
+This version introduced a backwards incompatible change to the Session API. 
```

Comment:
```suggestion
Consul 1.7.0 and 1.7.1 introduced a backwards incompatible change to the Session API. 
```

Reply:
The change was introduced in 1.7.0.

So how about:
```suggestion
Consul 1.7.0 introduced a backwards incompatible change to the Session API. 
```